torch_frame/datasets/huggingface_dataset.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import annotations
+
+import pandas as pd
+
+import torch_frame
+from torch_frame import stype
+from torch_frame.utils.split import SPLIT_TO_NUM
+
+
+class HuggingFaceDatasetDict(torch_frame.data.Dataset):
+    r"""Load a Hugging Face :obj:`datasets.DatasetDict` dataset
+    to a :obj:`torch_frame.data.Dataset` with pre-defined split information.
```

Comment:
Can you follow https://github.com/pyg-team/pytorch-frame/blob/master/torch_frame/datasets/tabular_benchmark.py? Add a list of dataset and stats.

Reply:
The datasets from HF is "dynamic" that user can push their own datasets. Now there are more than 80k datasets that I think it's hard to list them and their stats.